lib/make/makefile.bowtie2

```diff
@@ -95,6 +102,11 @@ all_interleaved_sorted_bams: $(subst .int.fastq.gz,.int.bowtie2.sorted.bam,$(wil
 	bowtie2 $(BOWTIE_ALIGN_OPTS) -x $(BOWTIE_INDEX) --interleaved $(word 1,$^) 2> $@.err | samtools view -Sb | samtools sort > $@
 	@echo "`date +"%Y%m%d %H:%M:%S"`: DONE" >> $@.makecall
 
+%.int.bowtie2.sorted.bam: %.int.fasta
```

Comment:
Always specify fasta files as `.fna` or `.faa` and, preferably, gzipped.

Reply:
Done.